index.html

```diff
@@ -1,70 +1,277 @@
+          ><img
+            src="./imgs/hamburguer-menu/hamburguer-menu-icon-enabled.png"
+            alt="Hamburguer Button"
+        /></a>
+      </nav>
+    </header>
+    <main class="background-image">
+
+      <section class="name-container">
+        <h1>
+          I'm Luis.
+          <br />
+          Glad to see
+          <br />
+          you!
+        </h1>
+        <!-- Large Text Section -->
+        <p class="first-paragraph">
+          I’m a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <h2>LET'S CONNECT</h2>
+        <!-- Social Media Section -->
+        <ul class="social-media1">
+          <li>
+            <a href="https://twitter.com/otromascomotu">
+              <img src="./imgs/icons/Twitter.svg" alt="twitter logo" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/luis-laverde-developer/">
+              <img src="./imgs/icons/Linkedin.svg" alt="linkedin logo" />
+            </a>
+          </li>
+          <li>
+            <a href="https://medium.com/@otromascomotu">
+              <img src="./imgs/icons/Medium.svg" alt="medium logo" />
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/Otromascomotu">
+              <img src="./imgs/icons/Github.svg" alt="github logo" />
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="./imgs/icons/Angelist.svg" alt="angellist logo" />
+            </a>
+          </li>
+        </ul>
+      </section>
+      <!-- Works Section -->
+      <section class="works-container">
+        <div class="card">
+          <div class="card-one">
+            <img
+              class="snapshot-portfolio"
+              src="./imgs/snapshot-portfolio/snapshot-portfolio1.svg"
+              alt="general portfolio image"
+            />
+          </div>
+          <div class="card-two">
+            <h3 class="tonic-title">Tonic</h3>
+            <div class="three-elements-row">
+              <span class="canopy-title"> CANOPY </span>
+              <div class="circle"></div>
+              <span class="back-end-title"> Back End Dev </span>
+              <div class="circle"></div>
+              <span class="year-title"> 2015 </span>
+            </div>
+            <p class="daily-selection-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="categories">
+              <li class="three-elements">HTML</li>
+              <li class="three-elements">CSS</li>
+              <li class="three-elements">Javascript</li>
+            </ul>
+            <button type="button" class="button1">See project</button>
```

Comment:
- [ ] **[OPTIONAL]** I would like to kindly suggest **adding effects to your buttons**, such as `enabled, pressed, hover, and disabled` states. By incorporating these effects, your project will not only look better but **also align more closely with the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122).** This small enhancement can make a big difference in the overall visual appeal. 🤓

   ![image](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/107952724/c4215312-348e-45ff-bb4f-cd3b95d9d151)

Reply:
yes, I'll add them soon!